packages/core/src/lib/TwaManifest.ts

```diff
@@ -45,6 +45,18 @@ export function asDisplayMode(input: string): DisplayMode | null {
   return DISPLAY_MODE_VALUES.includes(input) ? input as DisplayMode : null;
 }
 
+const ORIENTATION_VALUES = ['any', 'natural', 'landscape', 'portrait', 'portrait-primary',
```

Comment:
Could you add a comment linking to the web manifest spec for orientation values?

Reply:
This is actually from android-browser-helper: https://github.com/GoogleChrome/android-browser-helper/blob/alpha/androidbrowserhelper/src/main/java/com/google/androidbrowserhelper/trusted/LauncherActivityMetadata.java#L191-L216. I'll add a link to that.

Reply:
done.